### PR TITLE
Wizard recipe: TtH-v0.1.0

### PR DESCRIPTION
**Pull request by jverzani**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: TtH
* Version: v0.1.0

@staticfloat please review and merge.

### CODE REVIEW

File: T/TtH/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sources = [
+    ArchiveSource("http://hutchinson.belmont.ma.us/tth/tth-noncom/tth_C.tgz", "83c1f39fbf3377fb43e3d01d042302fa91f8758aa9acc10e22fe8af140f0126c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd tth_C
+gcc -o tth tth.c
+cp tth /usr/local/bin
```
Comment (giordano) [inline T/TtH/build_tarballs.jl:18]: This should work on all platforms:
```suggestion
cd $WORKSPACE/srcdir/tth_C
mkdir -p ${bindir}
gcc -o ${bindir}/tth${exeext} tth.c
```

File: T/TtH/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    
+]
```
Comment (giordano) [inline T/TtH/build_tarballs.jl:28]: I assume you want to define an [`ExecutableProduct`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilderBase.ExecutableProduct):
```suggestion
products = [
    ExecutableProduct("tth", :tth),   
]
```

Comment (jverzani) [reply T/TtH/build_tarballs.jl]: Thank you! I *very* much appreciate the help here.

**Comment by giordano** [comment]
Ok, it's all green now, let's go! Thanks for the contribution!

**Pull request merged**